docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: "Getting started with Grafana Mimir using the Helm chart"
+menuTitle: "Getting started using the Helm chart"
+description: "Learn how to get started with Grafana Mimir using the Helm chart."
+weight: 20
+---
+
+# Getting started with Grafana Mimir using the Helm chart
+
+The Helm chart allows you to configure, install, and upgrade Grafana Mimir within a Kubernetes cluster.
```

Comment:
This sentence serves to provide some content and context, to avoid having an empty bloated section.

We can add a homepage link to Helm to make this more valuable.